indico/web/templates/footer.html

```diff
@@ -4,7 +4,7 @@
             <a href="https://getindico.io">Indico</a>
         {%- endset -%}
         <div class="flexrow f-j-space-between">
-            <div class="flexrow f-a-center">
+            <div class="flexrow f-a-center f-self-stretch">
```

Comment:
I'm curious, why do we need to stretch the outer elements for this? Shouldn't space-between on the container already spread them evenly?

Reply:
If all items have `flex-grow` set to `1`, the remaining space in the container will be distributed equally to all children. In this case we have 3 children and we want the middle one to be at the exact center. So distributing the remaining space to the first and third child, we make sure that the 2nd one will always be in the middle.

Reply:
Even if the third element has no content ;)